package.json

```diff
@@ -34,6 +34,7 @@
     "@angular-devkit/schematics": "~9.1.0",
     "@docusaurus/core": "^2.0.0-alpha.50",
     "@docusaurus/preset-classic": "^2.0.0-alpha.50",
+    "@nrwl/cypress": "9.2.2",
```

Comment:
I understand this is required, but let's have a discussion about it and also why we install dependencies in our workspace that our plugins schematic's install (e.g. `vue`).

Reply:
Added cypress ^4.1.0